### ISSUE TITLE
Wrong colormap spec for HTML in documentation

### ISSUE THREAD

**Issue by f380cedric** [pgf-tikz/pgfplots#482]
Hello 👋,

In the _Colormap Input Format Reference_ (p. 196) https://github.com/pgf-tikz/pgfplots/blob/e4650af133358943881aee83e607f3567ff2d0a4/doc/latex/pgfplots/pgfplots.reference.markers-meta.tex#L915-L941
`HTML` is defined as `rbg` but instead of decimal, it's hex.
However, the package does not seems to accepts `HTML=(XX,XX,XX)` but `HTML=(XXXXXX)` (IMO, the latter format is better).

**Comment by Mo-Gul** [comment]
You are right. The description here is a bit short/imprecise. I think it is much better at

https://github.com/pgf-tikz/pgfplots/blob/e4650af133358943881aee83e607f3567ff2d0a4/doc/latex/pgfplots/pgfplots.reference.3dplots.tex#L1696-L1729

It should be in sync (again). At least there should be a link to the above position ...

(I didn't check yet, if these position ever were in sync.)
